2020-05-11 19:08:11,005 - coreapp - ERROR - CrechePrincipal matching query does not exist.
Traceback (most recent call last):
  File "/home/zigama/PycharmProjects/creche/coreapp/controller/report_controller.py", line 78, in get_principal
    result = service.get_principal(email = user.get('mail'))
  File "/home/zigama/PycharmProjects/creche/coreapp/service/principal_service.py", line 108, in get_principal
    if email: principal = CrechePrincipal.objects.get(email=email)
  File "/home/zigama/Projects/Python3/crechevenv/lib/python3.6/site-packages/django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/zigama/Projects/Python3/crechevenv/lib/python3.6/site-packages/django/db/models/query.py", line 408, in get
    self.model._meta.object_name
coreapp.appmodel.models.CrechePrincipal.DoesNotExist: CrechePrincipal matching query does not exist.
